replicate-to-sql-with-generic-events/lib/index.js

```diff
@@ -0,0 +1,48 @@
+const { PostgresWrapper, pgConfigDocker } = require('./postgres');
+
+class CorePlugin {
+  constructor() {
+    this.context = null;
+    this.config = {};
+    this.pipes = {
+      'generic:document:afterWrite': 'afterWrite',
+      'generic:document:afterDelete': 'afterDelete'
+    };
+  }
+
+  init(customConfig, context) {
+    this.config = Object.assign(this.config, customConfig);
+    this.context = context;
+    this.pg = new PostgresWrapper(pgConfigDocker);
+  }
+
+  getProperties(doc) {
+    const properties = Object.keys(doc._source).reduce((object, key) => {
+      if (key !== '_kuzzle_info') {
+        object[key] = doc[key];
+      }
+      return object;
+    }, {});
```

Comment:
The fact is that i want to properties to be erase from the object, if i set `properties._kuzzle_info = undefined;` it will be still present in the properties document witch is not what i want

Reply:
undefined values are not serializable and disappear from objects sent to either ES or postgre

But I agree it's a bit of a trick. If you want a more explicit way, just use `delete properties._kuzzle_info` instead, it's still more explicit than rebuilding the entire object using `Array.reduce`